### PR TITLE
optional logging - level and custom handler

### PR DESCRIPTION
**Pull request by rbdubz3**
added optional logging features:
- pass level into constructor
- pass custom handler into constructor
- add method to manipulate log level at runtime

### CODE REVIEW

File: lightify/__init__.py [hunk 1]
```diff
@@ -962,6 +968,14 @@ def _next_seq(self):
             self.__seq = (self.__seq + 1) % 256
             return self.__seq
 
+    def set_loglevel(self, level):
+        """ set the log level
+
+        :param level: log level (from logging module)
```
Comment (OleksandrBerchenko) [inline lightify/__init__.py]: For consistency you could change description to "logging.loglevel Enum"
```suggestion
        :param level: logging.loglevel Enum
```

Comment (OleksandrBerchenko) [reply lightify/__init__.py]: Otherwise looks good to me.  I will let you know when I test it with Home Assistant.

Thanks!

**Comment by rbdubz3** [comment]
sorry guys - GitHub neophyte - noticed another consistency change but not sure how to get the PR to pull the latest from my fork master .. I basically changed the actual param names 'log_level' to match in both constructor and set_loglevel method

**Comment by OleksandrBerchenko** [comment]
See https://github.com/tfriedel/python-lightify/pull/16/files - it looks like you have changed that successfully.  Does it display all your changes?

Thanks!

**Comment by rbdubz3** [comment]
@OleksandrBerchenko - yeah those look good .. think we are clean and consistent now with comments - thanks

**Comment by OleksandrBerchenko** [comment]
@tfriedel Hi!  Are you waiting for me to test it with Home Assistant logger, or am I waiting when it's merged first? :)

**Comment by rbdubz3** [comment]
@OleksandrBerchenko - curious on the support for 'scenes' - is there any way to invoke the built-in dynamic scenes from the Sylvania mobile app using the python library - Fireplace, Ocean, Candy, Polar Light, etc ?

**Comment by OleksandrBerchenko** [comment]
@rbdubz3 I don't know, sorry.

**Comment by tfriedel** [comment]
@OleksandrBerchenko I merged it now. Can you please test it with the HA logger?

**Comment by OleksandrBerchenko** [comment]
1\. In general, you don't need to pass HA log handler into Lightify class to control the logging.  Everything already works as expected.  For example, you may have in `configuration.yaml`:

```
logger:
  default: info
  logs:
    lightify: warning
```

2\. The only issue is that you can't set a less severe level than the one from Lightify:

```
self.__logger.setLevel(log_level)
```

In other words, if by default you have `log_level=logging.INFO`, you can't set it to `DEBUG` in `configuration.yaml`.  I have already fixed that in `osramlightify.py` by passing `logging.DEBUG` into `Lightify` constructor (using the new code from @rbdubz3).  See https://github.com/OleksandrBerchenko/home-assistant/blob/osramlightify/homeassistant/components/light/osramlightify.py.  So, now any level set in `configuration.yaml` will work as expected.

P.S. @tfriedel Let me know when you had a chance to go through my comments in https://github.com/tfriedel/python-lightify/pull/13.

When we finalize all the changes, you will need to trigger a new build on Pypi.

Thanks!

**Pull request merged**